app/src/main/java/nu/yona/app/api/model/AppMetaInfo.java

```diff
@@ -0,0 +1,46 @@
+package nu.yona.app.api.model;
+
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.widget.TextView;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import nu.yona.app.BuildConfig;
+import nu.yona.app.R;
+import nu.yona.app.YonaApplication;
+import nu.yona.app.ui.settings.SettingsFragment;
+import nu.yona.app.utils.AppConstant;
+import nu.yona.app.utils.AppUtils;
+
+import static nu.yona.app.ui.YonaActivity.getActivity;
+
+public class AppMetaInfo {
+
+    @SerializedName("operatingSystem")
+    @Expose
+    private String operatingSystem = YonaApplication.getAppContext().getString(R.string.operating_system);
```

Comment:
There is no value in using a resource for this. It's an enum value, not a translatable string.